canonical/ubuntu.com
# Meganav-rebase

## Done

- Final PR for the new meganav release

## QA

- Check that the appearance, functionality and content are the same in [this demo](https://ubuntu-com-12939.demos.haus/) as they are in [the original demo](https://ubuntu-com-12252.demos.haus/)
- Check the code diffs and see that that no new changes have been introduced (unless they are fixes)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4297
